docs/RELEASE_MANAGEMENT.md

```diff
@@ -4,11 +4,11 @@
 
 ### Upload metadata to AppStore
```

Comment:
Let's merge these two sections into one: Upload metadata and screenshots to AppStore.